website/sidebars.json

```diff
@@ -236,6 +235,7 @@
           "pact_broker/pacticipant_version_numbers",
           "pact_broker/advanced_topics/see_changes_pact",
           "pact_broker/advanced_topics/provider_verification_badges",
+          "pact_broker/advanced_topics/provider_verification_results",
```

Comment:
Not sure if it waas intended to have advanced topic page linked outside of the category though